### ISSUE TITLE
Use consistently "works, goods or services"

### ISSUE THREAD

**Issue by JachymHercher** [open-contracting/standard#1521]
We don't have a consistent expression for "stuff to be bought". We should. 

___ 

`item` is defined as:

> "A good, service, or work to be contracted."

and the `procurementCategory` contains the codes 'works', 'services', and 'goods and supplies'.

___ 

Currently, OCDS uses the following (a few of which might be specific for a reason, but most are probably not): 
- good and services (11 matches)
- goods, works or services (8 matches)
- supplies, services or works (3 matches) 
- goods, works and services (2 matches)
- goods and supplies (in the codename), goods or supplies (in the description) (2 matches)  
- goods and services (5 matches, out of which 4 deprecated)

The GPA uses "goods or services" and, rarely "goods and services". 

The EU directives use "works, supplies or services" and, rarely, "works, supplies and services". (I think the EU mentions works first because they are probably the most important/valuable segment. The reason why the EU can have works while the GPA does not is that works are just a combination of supplies and services.)

___ 

I think we should follow the following principles:
1. With the exception of the definition of 'goods' in `procurementCategory`, we should use just one of "goods" and "supplies". Goods is aligned with the GPA and seems more common, so I would stick to that. 
1. Since "works" are in OCDS, we should mention them along "goods" and "services" when we want to say "stuff to be bought". 
1. "Works" should either be first or last, but not between "goods" and "services" (because it's a bit of a different animal). 
1. "Goods" before "services" seems to be a typical way to phrase things, so we should stick to that.

With that in mind, we should either say "works, goods or services" or "goods, services or works". (With the "or" replaced by "and" depending on the context.) I would go for the former, but largely because I'm very much used to it from the EU (and because I like the argument that it is the most valuable part of procurement.)

**Comment by jpmckinney** [comment]
1. 👍
2. 👍 
3. That may be true, but putting it in the middle is much more common (in Google results at least)
4. 👍 that it appears somewhere before "services"

I know that works/infrastructure include the highest value procurements, but I can't actually find data on which category is responsible for the largest cumulative spend. It seems to me that goods *could* be more than works. We might also consider which is most frequent (rather than which is highest value), in which case I think it's goods.

**Comment by JachymHercher** [comment]
tldr: I guess numbers won't help us resolve this - what do you like / find the most standard? :) 

____ 

The EU's latest (2018) [public procurement indicators](https://ec.europa.eu/docsroom/documents/48156
) say the following (tables 10 - 15):

| EU 2018  | Value  | Number of contract award notices |
|----------|--------|----------------------------------|
| Works    | 147.89 | 23 979                           |
| Goods    | 115.59 | 65 979                           |
| Services | 180.77 | 89 322                           |

(I'm a bit skeptical about the value of services, because there tend to be some data troubles with framework agreements there, but on the other hand there are some efforts to compensate for it / clean up the data in the study, so who knows.)

___ 

Eyeballing the "sector overview" table in https://opentender.eu/ch/dashboards/market-analysis also doesn't say much more than "they look similarly large".

**Comment by jpmckinney** [comment]
I'd say the options are:

1. goods, works [conjunction] services
1. works, goods [conjunction] services
1. goods, services [conjunction] works

The other permutations occur very infrequently (per Google Search): "services" in the initial position and "works, services and goods".

2 and 3 meet the "works are a different animal" criterion.

I'm happy with any option. @yolile for another opinion?

**Comment by odscjen** [comment]
Note for once the final wording is agreed, the following fields will need changed/checked in release-schema.json (using 1.2-dev branch):
* `Tender.items`
* `Award.items`
* `buyer`
* `Item.unit`
* `Unit`
* `Contract.items`
* `Item.description`
* `Transaction.providerOrganization` (deprecated)
* `receiverOrganization` (deprecated)
* `Item`
Also `mainProcurementCategory` codelist order in schema.

The following codeslists will need changed/checked:
* procurementCategory.csv - order of codes and title of 'goods', currently "Goods and supplies"
* extendedProcurementCategory.csv - as for procurementCategory
* classificationScheme.csv - not the titles but the descriptions although will need checked incase they are official descriptions of the scheme (CPV, CPC, CUCOP)

Files that will need updated from the docs folder:
* schema/reference.md - ### Tender section, 2 instances
* schema/codelists.md - ### Document Type section, 1 instance
* primer/what.md - # What is the OCDS... section, 2 instances
* guidance/map/awards_contracts.md - ## Purchase orders section, 1 instance
* guidance/map/milestones.md - #### Delivery and payment data, 1 instance
* guidance/design/user_needs.md - ## Value for money in procurement section, 1 instance, ## Monitoring service delivery, 1 instance

**Comment by jpmckinney** [comment]
Thanks for cataloguing! I'll put a vote for goods, services and works as it groups "goods and services" separately from "works", and it's in alphabetical order, which can be a useful mnemonic for remembering the (somewhat arbitrary) order.

**Comment by odscjen** [comment]
I'd agree with that choice, alphabetical order is generally easier to remember. If @yolllie agrees I can get started on these changes?

**Comment by jpmckinney** [comment]
You can get started already - I just wanted another opinion and we've arrived at a reasonable choice.

**Comment by yolile** [comment]
Agree that "goods, services and works" sound good. In Spanish, it is most common to use "goods and services" (together and with "and") but "goods, services and works" sound good too.
